internal/pkg/logind/broker.go

```diff
@@ -48,11 +50,19 @@ func NewBroker(serviceSocketPath, clientSocketPath string) (*DBusBroker, error)
 		return nil, err
 	}
 
+	if err = selinux.SetLabel(serviceSocketPath, "system_u:object_r:dbus_service_socket_t:s0"); err != nil {
```

Comment:
should we do similar like standard distro, `system_dbus_var_run_t` :thinking:

Reply:
Unsure. No reason to either do as in reference policy nor do it differently. We can do as in ref though

Reply:
let keep what you have then.